Shared/Samples/Group layers together/GroupLayersTogetherView.GroupLayerListView.swift

```diff
@@ -116,4 +101,33 @@ extension GroupLayersTogetherView {
             }
         }
     }
+    
+    /// A toggle for changing a given layer's visibility.
+    private struct LayerVisibilityToggle: View {
+        /// The title of the toggle.
+        private let title: String
+        
+        /// The layer with the visibility to change.
+        private let layer: Layer
+        
+        /// A Boolean value indicating whether the layer's content is visible.
+        @State private var layerIsVisible: Bool
```

Comment:
The `LayerVisibilityToggle` already uses the `Layer` so this should just be called, `isVisible`.